src/components/Navbar/Drawer/Drawer.module.scss

```diff
@@ -2,7 +2,7 @@
 @use "src/styles/constants";
 
 .container {
-  background: var(--color-background-elevated);
```

Comment:
We'll probably lose some details if we use `faded`

with `faded`
![image](https://user-images.githubusercontent.com/12745166/143727121-8cedb2dc-1ba5-49bd-9284-13be2787b39d.png)


with `elevated`

![image](https://user-images.githubusercontent.com/12745166/143727126-33873eba-07b9-4ef8-81cf-1e5ac53523b5.png)

Reply:
the current loading animation colors didn't look good with the new bg:
![image](https://user-images.githubusercontent.com/3989750/144733912-4a6a53fd-02ea-46e5-bac9-a88d594de15f.png)

hmm, will try to change the skeleton color to look good while keeping the same bg.